### DISCUSSION TITLE
Default sorting of index pages does not work

### DISCUSSION THREAD

**Discussion by forgecodex** [laravel/nova-issues#6315] [Q&A]
- Laravel Version: latest
- Nova Version: latest
- PHP Version: latest
- Database Driver & Version:
- Operating System and Version: latest
- Browser type and version: latest
- Reproduction Repository: https://github.com/###/###

### Detailed steps to reproduce the issue on a fresh Nova installation:

`    
public static function indexQuery(NovaRequest $request, $query): Builder
    {
        return $query->orderBy('updated_at', 'desc');
    }

`

**Comment by webard** [comment]
Default ordering is applied by `defaultOrderings` method in Resource:

```php
public static function defaultOrderings($query)
{
   return $query->latest('updated_at');
}
```
